src/storage-client/src/controller.rs

```diff
@@ -1076,10 +1105,13 @@ where
                     )
                     .await;
 
+                let storage_dependencies = this.get_collection_dependencies(&description);
```

Comment:
Discussed this in person, but it seems like these dependencies also need to be fed into `install_read_capabilities`.

Reply:
pushed a commit that does this. h/t for finding this!